docs/src/manual/complex.md

```diff
@@ -125,7 +125,7 @@ julia> set_silent(model)
 julia> @variable(model, x[1:2]);
 
 julia> @constraint(model, (1 + 2im) * x[1] + 3 * x[2] == 4 + 5im)
-(1.0 + 2.0im) x[1] + (3.0 + 0.0im) x[2] = 4.0 + 5.0im
+(1 + 2im) x[1] + 3 x[2] = 4.0 + 5.0im
```

Comment:
I went for `1.0 + 2.0im` so that we didn't have differences depending on the left-hand and right-hand sides of expressions.

Reply:
That applies for `Float64` also then. We should apply our pretty printing to the rhs as well then

Reply:
`@constraint(m, 2x == 2)` would otherwise print as `2 x == 2.0` right ?

Reply:
Moved to a separate issue: https://github.com/jump-dev/JuMP.jl/issues/3339